src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -751,7 +770,33 @@ class GoogleExtensions(
       .lastOrError
   }
 
-  override val allSubSystems: Set[Subsystems.Subsystem] = Set(Subsystems.GoogleGroups, Subsystems.GooglePubSub, Subsystems.GoogleIam)
+  // TODO CORE-681: unit tests
```

Comment:
Are there more unit tests to be written in a later PR?

Reply:
oh whoops, I had that in there as a note to self, I meant to remove it when I wrote the tests. Will fix.